src/ThreadPool/ThreadPool.Tests/Tests.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Threading;
+using NUnit.Framework;
+
+namespace ThreadPool.Tests
+{
+    public class Tests
+    {
+        private const int CountOfTasks = 100;
+        private readonly IMyTask<int>[] _tasks = new IMyTask<int>[CountOfTasks];
+        private readonly Func<int>[] _functions = new Func<int>[CountOfTasks];
+        private MyThreadPool _threadPool;
+
+
+        [SetUp]
```

Comment:
```suggestion
        private MyThreadPool _threadPool;

        [SetUp]
```
Две пустые строки подряд обычно не ставят